Update `osu!stream` and fill in all the missing information on `osu!stream song list` (#8608)
ppy/osu-wiki: wiki/osu!stream/en.md

# osu!stream

*For the list of songs featured in the game, see: [osu!stream song list](/wiki/osu!stream/Song_List)*

![](img/Os-Logo.jpg "osu!stream logo")

> **osu!stream** is a special version of osu! made for handheld [iOS](http://en.wikipedia.org/wiki/IOS) and [Android](http://en.wikipedia.org/wiki/Android_%28operating_system%29) devices. It is available for download from the [App Store](https://apps.apple.com/us/app/osu-stream/id436952197) (iOS) and [Play Store](https://play.google.com/store/apps/details?id=sh.ppy.osustream&pli=1) (Android) free of charge.

Reviewer: ```suggestion
**osu!stream** is a special version of osu! made for handheld [iOS](http://en.wikipedia.org/wiki/IOS) and [Android](http://en.wikipedia.org/wiki/Android_(operating_system)) devices. It is available for download from the [App Store](https://apps.apple.com/us/app/osu-stream/id436952197) (iOS) and [Play Store](https://play.google.com/store/apps/details?id=sh.ppy.osustream&pli=1) (Android) free of charge.
```

> Unlike regular osu!, osu!stream features a completely new gameplay system, with new gameplay elements such as hold circles and connected beats. It does not support classic osu! beatmaps without manual conversion. Instead, maps are created by a special team of mappers to take advantage of the new elements and adhere to a [stricter level of standards](https://docs.google.com/document/d/1FYmHhRX-onR-osgTS6uHSOZuu_0JEbfRZePVySvvr9g).

Reviewer: also, `are created` -> `were created` to emphasize they are no longer being created?
```suggestion
Unlike regular osu!, osu!stream features a completely new gameplay system with unique elements such as hold circles and connected beats. It does not support classic osu! beatmaps without manual conversion. Instead, maps are created by a special team of mappers to take advantage of the new elements and adhere to a [stricter level of standards](https://docs.google.com/document/d/1FYmHhRX-onR-osgTS6uHSOZuu_0JEbfRZePVySvvr9g).
```

## History

> osu!stream's development began in 2011 as an attempt to introduce a version of osu! that can be played on-the-go following the previously abandoned [osu! iPhone](https://github.com/ppy/osu-iPhone) project. It was originally intended as an iOS-exclusive app, as Android's infrastructure at the time meant that audio latency issues on the platform were inevitable.

Reviewer: ```suggestion
osu!stream's development began in 2011 as an attempt to introduce a version of osu! that could be played on-the-go following the previously abandoned [osu! iPhone](https://github.com/ppy/osu-iPhone) project. It was originally intended as an iOS-exclusive app, as Android's infrastructure at the time meant that audio latency issues on the platform were inevitable.
```

> The game was first teased through [a cryptic YouTube video](https://www.youtube.com/watch?v=6Jyzgm_-joM) on June 2011. Following its public release a couple of months later, the behind-the-scenes of the game were elaborated in [osu!monthly \#5](https://osu.ppy.sh/community/forums/topics/59924) along with an interview.

Reviewer: ```suggestion
The game was first teased through [a cryptic YouTube video](https://www.youtube.com/watch?v=6Jyzgm_-joM) on June 2011. Following its public release a couple of months later, the behind-the-scenes of the game were elaborated in [osu!monthly \#5](https://osu.ppy.sh/community/forums/topics/59924) along with an interview with peppy.
```
